vmaas_sync/metrics.go

```diff
@@ -89,12 +89,20 @@ var (
 		Subsystem: "vmaas_sync",
 		Name:      "message_send_duration_seconds",
 	})
+
+	processesCount = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "backend_processes",
+	}, []string{"state"})
+
+	enableCyndiMetrics = utils.GetBoolEnvOrDefault("ENABLE_CYNDI_METRICS", false)
```

Comment:
Let's enable it by default. We are going to use it so we can only explicitly disable it when we really need.